### DISCUSSION TITLE
`reactpy-django`, now using a customized `mkdocs-material` theme!

### DISCUSSION THREAD

**Discussion by Archmonger** [squidfunk/mkdocs-material#5978] [Show and tell]
See preview: https://reactive-python.github.io/reactpy-django/latest/

![image](https://github.com/squidfunk/mkdocs-material/assets/16909269/cd95f57f-5c01-4de7-9dbd-fdc7d7badc15)

![image](https://github.com/squidfunk/mkdocs-material/assets/16909269/c18c971a-73b4-4e5d-8514-e905f88b8bfc)

![image](https://github.com/squidfunk/mkdocs-material/assets/16909269/00fec161-9751-488c-86b8-eec534ebaf28)

**Comment by squidfunk** [comment]
Oh wow, that looks super nice! Thanks for sharing!

**Reply by Archmonger** [reply]
Thank you for the amazing theme!

I look forward to the addition of a ["markdown render" template tag](https://github.com/squidfunk/mkdocs-material/discussions/5906#discussioncomment-6849731). Would let us replace homepage examples will actual codeblocks instead of screenshots.
